modules/twinklexfd.js

```diff
@@ -1076,21 +1244,24 @@ Twinkle.xfd.callbacks = {
 			var initialContrib = pageobj.getCreator();
 			var params = pageobj.getCallbackParameters();
 
+			// Also notify the user who owns the subpage if they are not the creator
+			params.userspaceOwner = mw.config.get('wgTitle').indexOf('/') === -1 ? mw.config.get('wgTitle') : mw.config.get('wgTitle').substring(0, mw.config.get('wgTitle').indexOf('/'));
```

Comment:
```suggestion
			params.userspaceOwner = mw.config.get('wgTitle').split('/')[0];
```

Reply:
Even better: this was presumably added before `wgRelevantUserName` was available, we should just use that.

Reply:
Also tossed in a `mw.config.exists('wgRelevantUserName')` for the `notifyuserspace` option, should now ensure that the user actually exists...